apps/web/components/ui/avatar/UserAvatar.tsx

```diff
@@ -0,0 +1,19 @@
+import { getUserAvatarUrl } from "@calcom/lib/getAvatarUrl";
+import type { User } from "@calcom/prisma/client";
+import { Avatar } from "@calcom/ui";
+
+type UserAvatarProps = Omit<React.ComponentProps<typeof Avatar>, "alt" | "imageSrc"> & {
+  user: Pick<User, "organizationId" | "name" | "username">;
+  /**
+   * Useful when allowing the user to upload their own avatar and showing the avatar before it's uploaded
+   */
+  imageSrc?: string | null;
```

Comment:
I find keeping imageSrc a little bit confusing, is there a case where we don't want to display the user's avatar when using UserAvatar component ?

wouldn't we want to just use the Avatar component if it is to pass an imageSrc props ?